src/main/java/com/fasterxml/jackson/databind/type/TypeBindings.java

```diff
@@ -325,7 +353,17 @@ public List<JavaType> getTypeParameters()
         if (_types.length == 0) {
             return Collections.emptyList();
         }
-        return Arrays.asList(_types);
+        List<JavaType> list = Arrays.asList(_types);
+        if (list.contains(null)) {
```

Comment:
problem is that Arrays.asList does not copy. A loop copy might be slightly faster if a null is present, but if there's no wildcard, it will surely be slower.

Reply:
Ok. That makes sense.